README.md

```diff
@@ -658,8 +47,8 @@ This table describes the list of available parameters for Helm Chart.
 | `resources.requests`                 | The resource requests for MarkLogic container                                                                  | `{}`                                 |
 | `nameOverride`                       | String to override the app name                                                                                | `""`                                 |
 | `fullnameOverride`                   | String to completely replace the generated name                                                                | `""`                                 |
-| `auth.adminUsername`                 | Username for default MarkLogic Administrator                                                                   | `admin`                              |
-| `auth.adminPassword`                 | Password for default MarkLogic Administrator                                                                   | ``    
+| `auth.adminUsername`                 | Username for default MarkLogic Administrator                                                                   | `admin`                              |
+| `auth.adminPassword`                 | Password for default MarkLogic Administrator                                                                   | ``    
 | `auth.walletPassword`                 | Password for wallet                                                                    | `` 
```

Comment:
Does this default to the admin password or does this have to be set?

Reply:
wallet-password have to be set. if not set docker script takes it as empty